django_elasticsearch_dsl/documents.py

```diff
@@ -63,25 +63,33 @@ def _format_index_language(cls, index, language):
         return '{0}-{1}'.format(index, language)
 
     @classmethod
-    def get_index_name_for_language(cls, index, language):
+    def _format_index_name(cls, index, language):
+        language_formatted_index = cls._format_index_language(index, language)
+        if settings.ELASTICSEARCH_DSL_ENV_PREFIX_ENABLED:
+            return '{0}-{1}'.format(settings.DEPLOYMENT_ENVIRONMENT, language_formatted_index)
```

Comment:
Rather than fixing prefix on development env, make this configurable and generic.
Add option for suffix as well.